src/main/java/net/snowflake/client/jdbc/SnowflakeChunkDownloader.java

```diff
@@ -284,6 +284,7 @@ public SnowflakeChunkDownloader(SnowflakeResultSetSerializableV1 resultSetSerial
   /** Submit download chunk tasks to executor. Number depends on thread and memory limit */
   private void startNextDownloaders() throws SnowflakeSQLException {
     long waitingTime = BASE_WAITING_MS;
+    long retry = 0;
```

Comment:
nit: rename it more specifically what kind of retry it is.

Reply:
renamed!